rocAL/rocAL/source/pipeline/master_graph.cpp

```diff
@@ -1493,163 +783,6 @@ bool MasterGraph::no_more_processed_data()
     return (_output_routine_finished_processing && _ring_buffer.empty());
 }
 
-MasterGraph::Status
-MasterGraph::copy_out_tensor_planar(void *out_ptr, RocalTensorFormat format, float multiplier0, float multiplier1,
```

Comment:
this was required for supporting planar output from the pipeline (like for CFar10 data set). Is this handled in tensor pipeline?

Reply:
Yes this is handled in tensor pipeline, some of our trainings require planar output i.e NCHW we are able to return a planar output from rocAL and normalization would happen in RPP.